style_guides/js_style_guide.md

```diff
@@ -344,92 +476,69 @@ if (a == '') {
 
 ## Only use ternary operators for small, simple code
 
-And **never** use multiple ternaries together
-
-*Right:*
+And *never* use multiple ternaries together
```

Comment:
Let's explain why:

```
And *never* use multiple ternaries together, because they make it more difficult to
reason about how different values flow through the conditions involved. Instead,
structure the logic for maximum readability.
```